test-bench-utils/lib/routes.js

```diff
@@ -138,5 +138,11 @@ module.exports = {
     products: ['Protect'],
     inputs: ['input'],
     sinks: sinks.xxe
+  },
+  paramPollution: {
+    base: '/parampollution',
+    name: 'HTTP Parameter Pollution / Cache Controls Missing',
+    link: 'TODO',
```

Comment:
I couldn't find an OWASP page for this vulnerability, so wasn't sure what to link.  maybe this slidedeck? https://owasp.org/www-pdf-archive/AppsecEU09_CarettoniDiPaola_v0.8.pdf

Reply:
I like that